### DISCUSSION TITLE
How to set a custom icon?

### DISCUSSION THREAD

**Discussion by jonygomes06** [hoodie/notify-rust#248] [Q&A]
I cant find info on how to set a custom icon for the notification.

How can i set the icon to the path of, for example, "C:\this\is\the\path\image.png"?

Is there any problem in using .png? or is it necessary to use .ico?

**Comment by rctlmk** [comment]
On Windows, you need to set an AUMID for a custom icon. Take a look at https://github.com/hoodie/notify-rust/issues/219#issuecomment-2323375004 and [this MSDN article](https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/send-local-toast-other-apps).
